starkliup/starkliup

```diff
@@ -91,6 +111,20 @@ install() {
     echo "Successfully installed starkli ${yellow}${LATEST_TAG}${normal}"
 }
 
+download_release_file() {
+    echo "Downloading release file from GitHub..."
+    if command -v curl >/dev/null 2>&1; then
+        http_code="$(curl -# -L "$FILE_URL" -o "$TEMP_FILE_NAME" --write-out "%{http_code}")"
```

Comment:
Can we make all variables (except the color codes) `SCREAMING_SNAKE_CASE`?

Reply:
I would say that it can be interesting to keep local variable in `snake_case`. But if you prefer, we then may switch everything to `SCREAMING_SNAKE_CASE`, a not local variables.

Anyway any variable declared is "scoped" to the current script execution. But IMHO it still interesting to know when a variable is local. :+1:

Reply:
Works for me.